lib/goals/build.js

```diff
@@ -25,11 +25,15 @@ const projectArchiver = require('../project-archiver');
 const buildConfigurator = require('../build-config');
 const imageStreamConfigurator = require('../image-stream');
 const binaryBuild = require('../binary-build');
+const helpers = require('../helpers');
 
 module.exports = async function build (config) {
   // archive the application source
   await projectArchiver.archiveAndTar(config);
 
+  // Removes the Dockerfile created by nodeshift.
+  await helpers.removeDockerfile(config.projectLocation);
```

Comment:
I'm not comfortable with this approach of removing the Dockerfile here in this location but it needs to happen after the tar creation if we decide to create the file on the disk. Or we can add this a documentation for the user that he will end up with a new file in their git repository after that (?)